test/reporters/base.spec.js

```diff
@@ -507,21 +507,34 @@ describe('Base reporter', function () {
     );
   });
 
-  describe('when reporter output immune to user test changes', function () {
-    var baseConsoleLog;
+  it('should list all the errors within an AggregateError', function () {
+    var err1 = new Error('1');
+    var err2 = new Error('2');
+    var aggErr = new AggregateError([err1, err2], '2 errors');
```

Comment:
[Testing] A few more test cases that should be covered...

* An `AggregateError` that only has one error
* An `AggregateError` where the constructor receives something that isn't an `Error` - which is possible per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError/AggregateError#errors

Reply:
I went ahead and added test cases for these.